project/Version.scala

```diff
@@ -5,7 +5,7 @@ object SpinalVersion {
   val isDev = true
   val isSnapshot = false
   private def snapshot = if (isSnapshot) "-SNAPSHOT" else ""
-  private val major = "1.7.2"
+  private val major = "1.7.3"
```

Comment:
Why is it modified in this PR?

Reply:
I think this modification triggers something could make #1009 not work at the head version. It might need caches to be cleaned and rebuild again.
How about fix thins major value to "dev" in dev branch and modify it in master branch?
@Dolu1990